test/Java8andUp/src_80/org/openj9/test/jdk/internal/agent/Test_FileSystem.java

```diff
@@ -0,0 +1,130 @@
+package org.openj9.test.jdk.internal.agent;
+
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.annotations.Test;
+import org.testng.AssertJUnit;
+import org.testng.log4testng.Logger;
+import sun.management.FileSystem;
+import java.io.File;
+import java.io.IOException;
+
+@Test(groups = { "level.sanity" })
+public class Test_FileSystem {
+	private static Logger logger = Logger.getLogger(Test_FileSystem.class);
+
+	private static boolean isWindows() {
+		String osName = System.getProperty("os.name");
+		if ((null != osName) && osName.startsWith("Windows")) {
+			return true;
+		}
+		return false;
+	}
+	
+	private static void resetPermissions(File file) {
+		file.setReadable(false, false);
+		file.setWritable(false, false);
+	}
+	
+	private static void runTest(FileSystem fs, File file, boolean expectedResult) {
+		try {
+			boolean result = fs.isAccessUserOnly(file);
+			logger.debug("Test passed: " + (expectedResult == result));
+			AssertJUnit.assertEquals(expectedResult, result);
+		} catch(IOException e) {
+			AssertJUnit.fail();
+		}
+	}
+	
+	/**
+	 * @tests sun.management.FileSystem#isAccessUserOnly(java.io.File f)
+	 */
+	public static void test_fileHasUserAccessOnly() {
+		/* method is not supported on windows */
+		if (isWindows()) {
+			logger.debug("skipping  test_fileHasUserAccessOnly on Windows");
+			return;
+		}
+		
+		FileSystem fs = FileSystem.open();
+		File file = new File("test-file-name");
```

Comment:
@smlambert concerns with this file created for testing? I've made sure to delete it at the end of each test.

Reply:
Creating a file is not an issue (as you clean it up after).  There are some other test changes that we will ask for on this PR, I've added @llxia for a review.